arm-customer-insights/2016-01-01/examples/RelationshipLinksDelete.json

```diff
@@ -0,0 +1,17 @@
+{
+	"parameters": {
+		"hubName": "sdkTestHub",
+		"resourceGroupName": "TestHubRG",
+		"relationshipLinkName": "Somelink",
+		"api-version": "2016-01-01",
+		"subscriptionId": "subid"
+	},
+	"responses": {
+		"200": {
+			
+		},
+		"200": {
```

Comment:
duplicate response codes are not allowed. Moreover, there should be an example for 202 response.